src/app/stacks/multistack_robocup_ssl.cpp

```diff
@@ -84,9 +86,11 @@ MultiStackRoboCupSSL::MultiStackRoboCupSSL(RenderOptions * _opts, int cameras) :
       *global_field);
 
   //add parameter for number of cameras
-  createThreads(cameras);
-  unsigned int n = threads.size();
-  for (unsigned int i = 0; i < n;i++) {
+  int numNormalCameraThreads = 4;
```

Comment:
This is fine for now for testing, but we really should move this to a config param rather than a hard-coded constant.

Reply:
where would you put such a parameter? Based on this number, the camera tabs and config tree are populated. Doing this during runtime would probably be a bit more effort.

Reply:
Config file or command-line params seem like reasonable compromises. Adding it to the GUI parameter tree will require some serious refactoring since this is required during initialization.
Again, it's fine for now.

Reply:
I've created a new PR: #127 

Regarding this parameter: I've added a cli parameter to set the number of camera threads. There is one issue though: The VarTree does not delete any nodes automatically. So there will be cameras left, if running with less camera threads then before. There will still be only the given number of threads and tabs, but more elements in the config tree...